packages/client/src/client/notificationsUtils.ts

```diff
@@ -40,3 +44,15 @@ export const processCreditQuote = (quote: PricedQuoteDetails) => {
     )} @ $${formatNumber(quote.state.payload)}`,
   }
 }
+
+export const processRFQRequestConfirmation = ({
```

Comment:
let's call this `processCreditRfqCreated` ... and please amend my earlier `processCreditAccepted` to `processCreditRfqAccepted`